bin/spack

```diff
@@ -176,7 +176,7 @@ def main():
 
 if args.profile:
     import cProfile
-    cProfile.run('main()', sort='tottime')
+    cProfile.run('main()', sort='time')
```

Comment:
I'll rebase eventually. I wanted to fix another command in the documentation anyway.